workshops/book-dash/meeting-notes/2023-09-05-WG.md

```diff
@@ -0,0 +1,57 @@
+# Book Dash [Provisional] Working Group Meeting 2023-09-05
+
+## Attendees 
+* Arielle Bennett
+* Emma Karoune 
+* (Apologies) Esther Plomp
+* (Apologies) Anne Lee Steele
+* (Apologies) Alex Araujo Alvarez
+
+## Agenda 
+* Working Group vs Planning Committee - governance and set up 
+* Applications for Nov 2023 Planning Committee
+* Activities for next Book Dash - discussion
+* Book Dash Hubs Set Up and Guidance
+
+## Decisions & Actions
+* Set up Working Group & separate Planning Committee in Nov following subsequent Book Dash - recommendation to core delivery team
+* Invite current and previous Planning Committee members to apply to WG in Nov - recommendation to core delivery team 
+* Accept all applications submitted to be part of this Planning Committee - recommendation to core delivery team
+* (Arielle) Upload notes to GitHub repo
+* (Arielle) communicate recommendations to core delivery team 
+
+## Notes
+* Book Dash working group vs planning committee team
+    * Assuming that we would still have RPM and RCM on planning committee, do need insight into operations and what is possible 
+    * (Recommendation) Planning committee - invite previous attendees to apply for next round as part of the follow up to next time 
+    * Working group - how are people selected/or how to do they apply? 
+        * Recommendations for basic criteria
+            * Have attended a Book Dash 
+            * [desirable] Have previously been on the planning committee at least once 
+        * Term length for working group: Two years? Would balance effort on elections/selections, with keeping the WG fresh 
+        * Could revisit options around renewing terms later 
+        * How to engage new people in between rounds? Planning committee/hubs? 
+        * How to manage people who **only** engage as part of the Book Dash? (Q for others as well)
+* Review applications to become part of the planning committee. Arielle has the access
+    * EK & AB would recommend accepting all to join the planning committee for this round - with the expectation that after the Book Dash some will join the Working Group
+    * Logic of this is due to timings an d lack of governanced docs difficult to stand up the WG right now 
+* Start brainstorming any activities that we would like to offer / to start quoting for the Book Dash this round
+    * Discussion around Scriberia images are sometimes duplicated and we struggle to get them into the turing Way itself - should we include them this time or skip and focus on imagathon?
```

Comment:
I think it would be important to keep Scriberia in - but to perhaps indeed add a imagathon session or two so that the images actually get added to the book.

Reply:
+1, though I also think that it's worth thinking alongside the infrastructure folks if we need a different kind of infrastructure for these images than simply uploading to Zenodo (as they are overwhelmingly useful and important for the community and for projects that extend beyond it). Can we have a searchable database for these images that others can grab? Should these be in their own repository? i.e. like the Noun Project